python/kserve/kserve/storage/storage.py

```diff
@@ -339,17 +339,29 @@ def _download_hdfs(uri, out_dir: str):
                 root=config["HDFS_ROOTPATH"],
                 session=s
             )
+        count = 0
+        dest_file_path = ""
 
         # Check path exists and get path status
         # Raises HdfsError when path does not exist
         status = client.status(path)
 
         if status["type"] == "FILE":
             client.download(path, out_dir, n_threads=1)
+            count = count + 1
+            file_name = path.rsplit("/", 1)[-1]
+            dest_file_path = f"{out_dir}/{file_name}"
         else:
             files = client.list(path)
             for f in files:
                 client.download(f"{path}/{f}", out_dir, n_threads=int(config["N_THREADS"]))
+                count = count + 1
```

Comment:
count can be simplified to `len(files)` instead of counting each loop here

Reply:
Thanks for the suggestion. It is done :).